### ISSUE TITLE
[Docs]: verifiedpermissions_schema

### ISSUE THREAD

**Issue by HanojHanu** [hashicorp/terraform-provider-aws#36090]
### Documentation Link

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/verifiedpermissions_schema

### Description

which one is correct to use, to get policy store id ?

``` tf
aws_verifiedpermissions_policy_store.create_policy_store.policy_store_id

# or

aws_verifiedpermissions_policy_store.create_policy_store.id
```

### Would you like to implement a fix?

None

**Comment by justinretzolk** [comment]
Hey @HanojHanu 👋 Generally speaking, we ask that issues in this repository be scoped to reporting bugs or requesting new features, and that questions be raised at one of our community resources such as the [AWS Provider forum](https://discuss.hashicorp.com/c/terraform-providers/tf-aws/33). That said, with many resources, the `id` (which is an identifier that every resources or data source that Terraform manages has) is the same value as another attribute of the resource -- this is one of those cases ([here](https://github.com/hashicorp/terraform-provider-aws/blob/c8cabe351eb587520af63418578edf5b9fc443e9/internal/service/verifiedpermissions/policy_store.go#L119) is a link to the relevant line of code that sets this). This is usually a result of following good [provider design principles](https://developer.hashicorp.com/terraform/plugin/best-practices/hashicorp-provider-design-principles#resource-and-attribute-schema-should-closely-match-the-underlying-api) and attempting to match the underlying API as closely as possible.

In other words, `id` is an identifier for Terraform that can be used in interpolations, and `policy_store_id` more closely matches the underlying API, and so is included as an attribute, but either may be used. I would generally recommend using the more descriptive of the two, but it comes down to preference.
